### DISCUSSION TITLE
How may I supress one vulnerability for 100 projects?

### DISCUSSION THREAD

**Discussion by workingoctopus** [DependencyTrack/dependency-track#2995] [Q&A]
We have 250+ projects and at least 100 of them have the same vulnerabilities in the same component. How may I suppress these 100 absolutely same vulns? Doing this manually is a very tedious job (because I have over 9000 more false positives).

**Comment by SaurabhAhuja1983** [comment]
@workingoctopus are you able to figure out the solution.

**Reply by workingoctopus** [reply]
I used https://github.com/nscuro/dtapac and it helped me :)

**Comment by nickthebeer** [comment]
> We have 250+ projects and at least 100 of them have the same vulnerabilities in the same component. How may I suppress these 100 absolutely same vulns? Doing this manually is a very tedious job (because I have over 9000 more false positives).

Have you looked into using the API to bulk update them? https://docs.dependencytrack.org/integrations/rest-api/
